django/db/backends/postgresql/base.py

```diff
@@ -257,6 +258,16 @@ def init_connection_state(self):
             if not self.get_autocommit():
                 self.connection.commit()
 
+        # Set the role on the connection. This is useful if the
+        # credential used to login is not the same as the role that
+        # owns database resources. This scenario can occur when using
+        # temporary credentials generated by an external system like
+        # Vault.
+        options = self.settings_dict["OPTIONS"]
+        if options.get("assume_role"):
```

Comment:
I'd probably do `if self.settings_dict["OPTIONS"].get("assume_role")` if it is short enough to pass the linters.

Reply:
I move this to one `.get` which I think looks cleaner. WDYT?